### DISCUSSION TITLE
Scout projectile prediction and Fov change through input

### DISCUSSION THREAD

**Discussion by g4503** [Fedoraware/Fedoraware#1133] [Ideas]
Title explains it.
Scouts throwables, like the guilloutine, madmilk and wrap assasin won't get corrected by the Aimbot.
And i'd preferr for Sniper to have a Fov that can be changed by inputs, rather than a slider, to be able to have an Fov of 0.2 or
sthe Fov keeps the same size from unscoped to zoomed in.

i have seen a thread talk about it but not mentioning the other weapons

**Comment by ghost** [comment]
the aim fov is based off of how much your viewangle has to change for you to get to x point, when zoomed your actual field of view is smaller, so it seems like your aim fov is larger, but it is actually the same amount.

you can have an aim fov of 0.2.

if aim fov's are to be changable by binds it will be with a feature that adds keybinds to all vars in the cheat.
